WooCommerce/WooCommerceUITests/Flows/MockDataReader.swift

```diff
@@ -10,22 +10,43 @@ class GetMocks {
         "outofstock": "out of stock"
     ]
 
-    static func getProductsMockDataContent(withFilename filename: String) -> Data {
-        let json = Bundle(for: ProductsTests.self).url(forResource: filename, withExtension: "json")!
+    var productName = [
```

Comment:
From looking at the code, I think this can be a constant?

```suggestion
    let productName = [
```

Using constants (`let`) by default is a useful hygiene practice that, when applied consistently, can help prevent the occasional bug due to code mutating values unbeknownst to the developer.